pkg/apis/elasticsearch/v1/elasticsearch_types.go

```diff
@@ -99,8 +99,8 @@ type TransportTLSOptions struct {
 	// and private key for generating node certificates.
 	// The referenced secret should contain the following:
 	//
-	// - `tls.crt`: The CA certificate in PEM format.
-	// - `tls.key`: The private key for the CA certificate in PEM format.
```

Comment:
Should we keep these 2 lines ?  My understanding is that `tls.crt` and `tls.key` are still required if `ca.key` is not provided.

Reply:
This is for the transport layer. I replaced the `tls.crt` `tls.key` keys (but offer backwards compatibility for existing users). Your statement is true for the HTTP layer.

Reply:
My bad, I missed the `TransportTLSOptions` in the diff header.